data/migrations/20191220192532_cars-schema.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('cars', tbl => {
+    tbl.increments('id');
+    tbl.string('VIN')
```

Comment:
regarding the casing inside sql and databases.  Convention is snake case (example `snake_case`).  I know that this file is js, but the strings passed in are turned into column names so you should convert them to snake case.